packages/react/package.json

```diff
@@ -25,11 +25,12 @@
     "lodash": "^4.17.15",
     "popper.js": "^1.15.0",
     "prop-types": "^15.7.2",
-    "react-fela": "^10.6.1",
-    "react-is": "^16.6.3"
```

Comment:
Is it intentional?

Reply:
`react-is` only appears to be used by `@fluentui/react-component-ref`.

Reply:
Ok, cool 👍